aicsimageio/writers/ome_tiff_writer.py

```diff
@@ -17,35 +17,38 @@
 
 
 class OmeTiffWriter:
-    """This class can take arrays of pixel values and do the necessary metadata creation to write them
-    properly in OME xml format.
+    def __init__(self, file_path, overwrite_file=None):
+        """
+        This class can take arrays of pixel values and do the necessary metadata creation to write them
+        properly in OME xml format.
+
+        Parameters
+        ----------
+        file_path
+            Path to image output location
+        overwrite_file
+            Flag to overwrite image or pass over image if it already exists.
+            None: (default) throw IOError if file exists
```

Comment:
I always wish True were the default

Reply:
It seriously should be 😬 more of a PITA than a useful safety net